packages/builder/src/builtin-tasks/migrate.ts

```diff
@@ -0,0 +1,59 @@
+import debug from "debug";
+import fsExtra from "fs-extra";
+import path from "path";
+import fs from "fs";
+
+import { task } from "../internal/core/config/config-env";
+import { BuilderError } from "../internal/core/errors";
+import { ERRORS } from "../internal/core/errors-list";
+import { runScriptWithAlgob } from "../internal/util/scripts-runner";
+import { TASK_MIGRATE } from "./task-names";
+import { glob } from "./util";
+import { RuntimeArgs } from "../types";
+import { runSingleScript } from "./run";
+
+export default function () : void {
+  const log = debug("builder:core:tasks:migrate");
+
+  task(TASK_MIGRATE, "Compiles and runs user-defined scripts from scripts directory")
+    .addPositionalParam(
+      "directory",
+      "A directory that contains js files to be run within builder's environment",
+      "scripts"
```

Comment:
According to the spec, the task should be called `deploy` and as an optional parameter, it should take list of files.

Reply:
You're trying to make it work as `run`. There is already a task like that.